File: src/app/models/sector.ts
export class Sector {
    constructor(
        private _ID: number,
        private _name: string,
        private _description?: string
    ) { }
    get ID() {
        return this._ID;
    }
    get name() {
        return this._name;
    }
    get description() {
        return this._description;
    }
}

File: Controllers/API_HRMS/Models/Sector.cs
using Microsoft.VisualBasic;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics.CodeAnalysis;

namespace API_HRMS.Models
{
    public class Sector
    {
        [Key]
        public int _ID { get; set; }
        public string _name{ get; set; }
        public int _departmentID { get; set; }
        public string? _description { get; set; }
    }
}
